client/src/components/client-page/index.jsx

```diff
@@ -0,0 +1,846 @@
+            <Grid container direction="row" alignItems="center">
+            {isEditMode ? (
+                <>
+                <Grid item xs={4}>
+                <Typography gutterBottom variant="body1" align="left">
+                  Phone Number
+                </Typography>
+              </Grid>
+              <Grid item xs={7.5}>
+                <Typography
+                  gutterBottom
+                  variant="body1"
+                  align="left"
+                  color={clientInfo.phone ? "black" : "#A9A9A9"}
+                >
+                  {clientInfo.phone
+                    ? clientInfo.phone
+                    : "Enter Phone Number..."}
+                </Typography>
+              </Grid>
+              <Grid item xs={0.5} id="info-card-icon">
+                <IconButton>
+                <ContentCopyIcon onClick={() => handleCopyClick(clientInfo.phone)} />
+                </IconButton>
+              </Grid>
+                </>
+              ) : (
+                <>
+                  <Grid item xs={1.5}>
+              <Typography gutterBottom variant="body1" align="left">
+              Phone Number
+              </Typography>
+            </Grid>
+            <Grid item xs={10}>
+            <TextField
+              value={editedPhone}
+              onChange={handlePhoneChange}
+              fullWidth
+              id="phone"
+              name="phone"
+              InputProps={{
+                inputComponent: TextMaskCustom,
+              }}
+            />
+            </Grid>
+                </>
+              )}
+            </Grid>
+          </Box>
+          <Divider variant="middle" />
+          <Box paddingTop={2} paddingBottom={1} paddingLeft={3}>
+            <Grid container direction="row" alignItems="center">
+            {isEditMode ? (
+                <>
+                 <Grid item xs={4}>
+                <Typography gutterBottom variant="body1" align="left">
+                  Status
+                </Typography>
+              </Grid>
+              <Grid item xs={8}>
+                <Typography variant="body1" align="left">
+                {clientInfo.status
+                    ? <Chip variant="filled" label={clientInfo.status} />
+                    : "Unknown"}
+                </Typography>
+              </Grid>
+                </>
+              ) : (
+                <>
+                  <Grid item xs={1.5}>
+                <Typography gutterBottom variant="body1" align="left">
+                  Status
+                </Typography>
+              </Grid>
+              <Grid item xs={10} sx={{
+        '& .MuiOutlinedInput-input': { display: 'flex', alignItems: 'flex-start' }
+      }}>
+                <Select 
+                fullWidth 
+                value={editedStatus} 
+                onChange={handleStatusChange}
+                renderValue={() =>
+                  editedStatus && (
+                    <Chip
```

Comment:
Don't let the user delete the status here. I'm not sure that the database supports an empty status atm.

Reply:
I changed it to a chip with label Undefined like on Figma